### ISSUE TITLE
Can we better organize the resources?

### ISSUE THREAD

**Issue by bjornmeansbear** [GD399-OSD/resources-readings#7]
Anyone want to think about this or work on this?

**Comment by devinhalladay** [comment]
Yeah I think we can just do some more headers & add descriptions. I can take on some of the work, but would be cool if everyone just edits their links in the README to include titles & descriptions.

I'll take a look at similar repositories (maybe some of the Learning Gardens repos) and see if they have a cool solution.

**Comment by bjornmeansbear** [comment]
No one person should need to do it all, if we can just collectively edit whatever we added, and then maybe split up as necessary all the things I seeded?

**Comment by devinhalladay** [comment]
yeah that sounds ideal. I'll edit my things next time I get a chance.
